engine/engine/src/engine_service.cpp

```diff
@@ -661,19 +661,8 @@ namespace dmEngineService
         return SendResourceData(request, name, extension, resource.m_Size, resource.m_SizeOnDisc, resource.m_RefCount);
     }
 
-    static bool DynamicTextureIteratorFunction(dmhash_t gui_res_id, dmhash_t name_hash, uint32_t size, void* user_ctx)
-    {
-        dmWebServer::Request* request = (dmWebServer::Request*)user_ctx;
-        const char* texture_name = dmHashReverseSafe64(name_hash);
-        const char* gui_name = dmHashReverseSafe64(gui_res_id);
-        char full_name[512];
-        dmSnPrintf(full_name, sizeof(full_name), "%s\n%s", gui_name, texture_name);
-        return SendResourceData(request, &full_name[0], "GuiDynamicTexture", size, 0, 1);
-    }
```

Comment:
Fix for reporting the correct memory usage.
Previously we kept the gui dynamic textures in the gui, but nowadays they're already stored as regular resources (and reported that way)